static/js/tools/timeline_util.ts

```diff
@@ -15,20 +15,50 @@
  */
 import axios from "axios";
 import { getUrlVars, setSearchParam } from "./dc";
-import { SEP } from "./statsvar_menu.tsx";
+import { SEP } from "./statsvar_menu";
+import { keys } from "../../webpack.config";
```

Comment:
what is this keys used for?

Reply:
Oops. I think it's added by mistake. I removed this line.